cmd/podman/common.go

```diff
@@ -374,7 +374,7 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 	)
 	createFlags.Int64(
 		"memory-swappiness", -1,
-		"Tune container memory swappiness (0 to 100) (default -1)",
+		"Tune container memory swappiness (0 to 100)",
```

Comment:
This says 0 to 100 and then has a default of -1 and that does not seem right.

Does -1 represent no limit?

Reply:
-1 == take the default setting from the system.